packages/components/src/theme/ThemeUtils.test.ts

```diff
@@ -333,6 +336,47 @@ describe('getDefaultBaseThemes', () => {
       },
     ]);
   });
+
+  /**
+   * This test is to ensure that the css variables in theme-dark and theme-light are in sync,
+   * to prevent the case where a css variable is added to one theme and not the other.
+   */
+  it('light and dark themes should contain the same css variables', () => {
+    const themeDarkDir = './packages/components/src/theme/theme-dark';
+    const themeLightDir = './packages/components/src/theme/theme-light';
+
+    function extractCssVariablesFromFiles(directory: string): Set<string>[] {
+      const files = fs.readdirSync(directory);
+      const fileVariables = [] as Set<string>[];
```

Comment:
I would set the type on the left side in this case as it is slightly stricter. `const fileVariables: Set<string>[] = []`